### PR TITLE
new update comapy

### PR DESCRIPTION
**Pull request by 7un7un**

### CODE REVIEW

File: frontend/src/Modules/Company/Components/Contact/Contact.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PhoneEnabledIcon from '@material-ui/icons/PhoneEnabled';
 import LanguageIcon from '@material-ui/icons/Language';
 import EmailIcon from '@material-ui/icons/Email';
 import LocationOnIcon from '@material-ui/icons/LocationOn';
-import { Http } from '../../Shared';
+import { Http } from '../../Shared/';
```
Comment (tanvuongduc) [inline frontend/src/Modules/Company/Components/Contact/Contact.js:7]: discard this

File: frontend/src/Modules/Company/Components/Document/Document.js [hunk 2]
```diff
@@ -1,45 +1,125 @@
...
+    getDocument = () => {
+        CompanyService.getCompanyByLocation()
+            .then(res => {
+                this.setState({
+                    company: res.data,
+                    document: res.data.documents
+                })
+            })
+            .catch((error) => {
+                console.error('Error:', error);
```
Comment (tanvuongduc) [inline frontend/src/Modules/Company/Components/Document/Document.js:41]: should show a notify to user when got err

File: frontend/src/Modules/Company/Components/Document/Document.js [hunk 3]
```diff
@@ -1,45 +1,125 @@
...
+        this.state.document.push(idDocumentUpload);
+        this.setState({
+            document: this.state.document
+        })
+
+        CompanyService.finishDocumentResult(this.state.document, this.state.company)
+            .catch((error) => {
+                console.error('Error:', error);
+            })
+    };
```
Comment (tanvuongduc) [inline frontend/src/Modules/Company/Components/Document/Document.js]: where is .then?

File: frontend/src/Modules/Company/Components/Document/Document.js [hunk 4]
```diff
@@ -1,45 +1,125 @@
...
+        if (event) {
+            const newDocument = this.state.document.filter((item) => item._id !== this.state.id_delete)
+            this.setState({ document: newDocument })
 
-    const showDocumentItem = dataDocument.map((event, index) => {
+            const _idDocument = newDocument.map(e => e._id)
+            CompanyService.finishDocumentResult(_idDocument, this.state.company)
+                .catch((error) => {
+                    console.error('Error:', error);
+                })
```
Comment (tanvuongduc) [inline frontend/src/Modules/Company/Components/Document/Document.js]: .then?

File: frontend/src/Modules/Company/Components/Note/ModalAdd.js [hunk 5]
```diff
@@ -8,7 +8,7 @@ const getModalStyle = () => {
     return {
         top: '35%',
         left: '50%',
-        width: '600px',
+        width: '50%',
         padding: '30px',
         borderRadius: '4px',
         position: 'absolute',
```
Comment (tanvuongduc) [inline frontend/src/Modules/Company/Components/Note/ModalAdd.js:14]: we dont use this any more. Should refactoring, move it to scss files

File: frontend/src/Modules/Company/Components/Note/Note.js [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+        this.state.note.push(item)
+        this.setState({
+            note: this.state.note
+        });
+
+        CompanyService.finishNoteResult(this.state.note, this.state.company)
+            .catch((error) => {
+                console.error('Error:', error);
+            })
+    };
```
Comment (tanvuongduc) [inline frontend/src/Modules/Company/Components/Note/Note.js]: then?

File: frontend/src/Modules/Company/Components/Note/_Note.scss [hunk 7]
```diff
@@ -0,0 +1,49 @@
+.note-content {
+    width: 100%;
+    height: auto;
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 50px;
+    padding: 30px;
+    border-radius: 4px;
+    background-color: white;
+    box-shadow: 0px 2px 1px -1px gray, 0px 1px 1px 0px gray, 0px 1px 3px 0px gray;
```
Comment (tanvuongduc) [inline frontend/src/Modules/Company/Components/Note/_Note.scss:10]: if u using card in material UI, all of this already have

File: frontend/src/Modules/Company/Components/Note/_Note.scss [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+        margin-top: 5px;
+        padding-top: 15px;
+        border-top: 1px solid #e6e9eb;
+        div {
+            .note-item-title {
+                width: 100%;
+                height: auto;
+                display: flex;
+                flex-wrap: wrap;
+                text-align: justify;
```
Comment (tanvuongduc) [inline frontend/src/Modules/Company/Components/Note/_Note.scss:30]: use flex box of material UI

File: frontend/src/Modules/Company/Components/Note/_Note.scss [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    padding: 30px;
+    border-radius: 4px;
+    background-color: white;
+    box-shadow: 0px 2px 1px -1px gray, 0px 1px 1px 0px gray, 0px 1px 3px 0px gray;
+    .note-header {
+        width: 100%;
+        height: auto;
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
```
Comment (tanvuongduc) [inline frontend/src/Modules/Company/Components/Note/_Note.scss:16]: use flex box of material UI

File: frontend/src/Shared/Components/Upload/Upload.js [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+		event.preventDefault();
+		event.target.reset();
+
+		const formData = new FormData();
+		formData.append(
+			"file",
+			this.state.selectedFile,
+			this.state.selectedFile.name
+		);
+		const upload = await axios.post("http://103.138.108.104:3000/upload", formData)
```
Comment (tanvuongduc) [inline frontend/src/Shared/Components/Upload/Upload.js]: what the hell

File: frontend/src/Shared/Services/UtilService.js [hunk 11]
```diff
@@ -1,4 +1,16 @@
+// import { REGEX_TEL } from '../../Constances/const'
+
 class UtilService {
+    _winOpen(path, target = '_blank') {
+        let url = "http://103.138.108.104:3000";
```
Comment (tanvuongduc) [inline frontend/src/Shared/Services/UtilService.js:5]: what the hell

**Review by tanvuongduc — CHANGES_REQUESTED** [review]

File: frontend/src/Modules/Company/Components/Contact/Contact.js (hunk 1 above)
Comment (7un7un) [reply frontend/src/Modules/Company/Components/Contact/Contact.js:7]: i will delete it

File: frontend/src/Modules/Company/Components/Document/Document.js (hunk 2 above)
Comment (7un7un) [reply frontend/src/Modules/Company/Components/Document/Document.js:41]: use .then() ?

File: frontend/src/Modules/Company/Components/Document/Document.js (hunk 3 above)
Comment (7un7un) [reply frontend/src/Modules/Company/Components/Document/Document.js]: i will add it

File: frontend/src/Modules/Company/Components/Note/ModalAdd.js (hunk 5 above)
Comment (7un7un) [reply frontend/src/Modules/Company/Components/Note/ModalAdd.js:14]: agree

File: frontend/src/Modules/Company/Components/Note/_Note.scss (hunk 7 above)
Comment (7un7un) [reply frontend/src/Modules/Company/Components/Note/_Note.scss:10]: agree

File: frontend/src/Shared/Components/Upload/Upload.js (hunk 10 above)
Comment (7un7un) [reply frontend/src/Shared/Components/Upload/Upload.js]: Trung trieu le write it, I'll fix it :v

**Pull request closed without merging**